dist/rpm/os-autoinst.spec

```diff
@@ -250,6 +250,9 @@ cd %{__builddir}
 
 %post openvswitch
 %service_add_post os-autoinst-openvswitch.service
+if test $1 -eq 1 ; then
+  %{_bindir}/dbus-send --system --type=method_call --dest=org.freedesktop.DBus / org.freedesktop.DBus.ReloadConfig 2>&1 || :
```

Comment:
So considering the conversation in #5419 will this fail in CI? And where is $1 defined?

Reply:
> So considering the conversation in #5419 will this fail in CI?

I guess this can't fail in CI because OBS CI checks only check the main package, not the openvswitch subpackage unlike openQA (see e.g. OBS checks in https://github.com/os-autoinst/openQA/pull/5352) showing OBS: openQA:openQA-client-test and OBS: openQA:openQA-test and such.

> And where is $1 defined?

I guess `$1` is defined in the RPM macro and the caller.

Reply:
Do we need to depend on `dbus-1-tools` for this line to work?

Reply:
yes - I added the `Requires(post)` depdency

Reply:
`$1` is `1` on install and `2` on update.

Reply:
@Martchus I added a requirement to `dbus-1` which requires `dbus-1-tools`